src/_locales/de/messages.json

```diff
@@ -35,6 +35,9 @@
   "context": {
     "message": "Im Kontextmenü anzeigen:"
   },
+  "pageActionOption": {
+    "message": "Weergeven in adresbalk:"
```

Comment:
I would use `Im Adressleiste anzeigen,` but if you are German that is fine. The word `Weergeven` is dutch.